internal/command/machine/update.go

```diff
@@ -86,6 +88,33 @@ func runUpdate(ctx context.Context) (err error) {
 		return
 	}
 
+	s := strings.Split(machineDiff, ",")
+	var str string
+	for _, val := range s {
+		_, foundDeletion := presenters.GetStringInBetweenTwoString(val, "-", ":")
+		_, foundAddition := presenters.GetStringInBetweenTwoString(val, "+", ":")
+		if foundAddition {
+			str += colorize.Green(val)
+		} else if foundDeletion {
+			str += colorize.Red(val)
+		} else {
+			str += val
+		}
+	}
+	fmt.Fprintln(out, "The following config is being updated")
+	fmt.Fprintln(out, str)
+
+	// interactive update //
+	name := false
+	prompt := &survey.Confirm{
```

Comment:
The prompt package has you covered: https://github.com/superfly/flyctl/blob/b9e83116ebbb045b94fa0bb25930f90ab4380797/internal/prompt/prompt.go#L125-L142